source/connect.txt

```diff
@@ -204,6 +204,15 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to A Different Deployment
+---------------------------------
+
+You can use the :method:`Mongo()` or the :method:`connect()` methods to 
```

Comment:
:method:`connect()` isn't rendering in staging.

Reply:
nvm, the manual doesn't use :method: to link to this for some reason. went with the workaround.